### PR TITLE
add hide contact information

### PR DESCRIPTION
**Pull request by sebest06**
Hello, i am adding a information to hide contacts from the roster.
The porpouse of this commit is had a roster more clean. if you like this can add to the project, thank you

### CODE REVIEW

File: src/main/res/values/strings.xml [hunk 1]
```diff
@@ -398,6 +399,7 @@
     <string name="sending_x_file">Sending %s</string>
     <string name="offering_x_file">Offering %s</string>
     <string name="hide_offline">Hide offline</string>
+    <string name="show_hide_contact">Show hide contacts</string>
```
Comment (licaon-kter) [inline src/main/res/values/strings.xml]: Maybe rephrase this to `Show hidden contacts`?

**Comment by licaon-kter** [comment]
Some pics?

You can hide multiple or one-by-one?

**Comment by iNPUTmice** [comment]
I don’t think this is the right fit for Conversations. You can a) just delete roster items if you don’t wan them b) usually you don’t open the contacts screen very often anyway plus it has a search.

**Comment by sebest06** [comment]
Ok, thanks!
